[2025-09-04T01:09:01.625+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: continuous_streaming_consumer.continuous_producer_consumer scheduled__2025-09-04T01:06:00+00:00 [queued]>
[2025-09-04T01:09:01.630+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: continuous_streaming_consumer.continuous_producer_consumer scheduled__2025-09-04T01:06:00+00:00 [queued]>
[2025-09-04T01:09:01.630+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2025-09-04T01:09:01.630+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1000
[2025-09-04T01:09:01.630+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2025-09-04T01:09:01.637+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): continuous_producer_consumer> on 2025-09-04 01:06:00+00:00
[2025-09-04T01:09:01.640+0000] {standard_task_runner.py:55} INFO - Started process 645 to run task
[2025-09-04T01:09:01.642+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'continuous_streaming_consumer', 'continuous_producer_consumer', 'scheduled__2025-09-04T01:06:00+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/dag_continuous_consumer.py', '--cfg-path', '/tmp/tmpx6d1gx6t']
[2025-09-04T01:09:01.643+0000] {standard_task_runner.py:83} INFO - Job 8: Subtask continuous_producer_consumer
[2025-09-04T01:09:01.654+0000] {logging_mixin.py:137} WARNING - /usr/local/lib/python3.9/site-packages/***/settings.py:249 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-09-04T01:09:01.668+0000] {logging_mixin.py:137} WARNING - /usr/local/lib/python3.9/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-09-04T01:09:01.688+0000] {task_command.py:388} INFO - Running <TaskInstance: continuous_streaming_consumer.continuous_producer_consumer scheduled__2025-09-04T01:06:00+00:00 [running]> on host a9c36e7e0f71
[2025-09-04T01:09:01.731+0000] {taskinstance.py:1507} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Klaus_Rezende
AIRFLOW_CTX_DAG_ID=continuous_streaming_consumer
AIRFLOW_CTX_TASK_ID=continuous_producer_consumer
AIRFLOW_CTX_EXECUTION_DATE=2025-09-04T01:06:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2025-09-04T01:06:00+00:00
[2025-09-04T01:09:01.735+0000] {conn.py:380} INFO - <BrokerConnection node_id=bootstrap-0 host=kafka:29092 <connecting> [IPv4 ('172.16.240.5', 29092)]>: connecting to kafka:29092 [('172.16.240.5', 29092) IPv4]
[2025-09-04T01:09:01.736+0000] {conn.py:1205} INFO - Probing node bootstrap-0 broker version
[2025-09-04T01:09:01.736+0000] {conn.py:410} INFO - <BrokerConnection node_id=bootstrap-0 host=kafka:29092 <connecting> [IPv4 ('172.16.240.5', 29092)]>: Connection complete.
[2025-09-04T01:09:01.739+0000] {conn.py:380} INFO - <BrokerConnection node_id=bootstrap-0 host=kafka:29092 <connecting> [IPv4 ('172.16.240.5', 29092)]>: connecting to kafka:29092 [('172.16.240.5', 29092) IPv4]
[2025-09-04T01:09:01.740+0000] {conn.py:1205} INFO - Probing node bootstrap-0 broker version
[2025-09-04T01:09:01.740+0000] {conn.py:410} INFO - <BrokerConnection node_id=bootstrap-0 host=kafka:29092 <connecting> [IPv4 ('172.16.240.5', 29092)]>: Connection complete.
[2025-09-04T01:09:01.838+0000] {conn.py:1267} INFO - Broker version identified as 2.5.0
[2025-09-04T01:09:01.838+0000] {conn.py:1268} INFO - Set configuration api_version=(2, 5, 0) to skip auto check_version requests on startup
[2025-09-04T01:09:01.842+0000] {conn.py:1267} INFO - Broker version identified as 2.5.0
[2025-09-04T01:09:01.842+0000] {conn.py:1268} INFO - Set configuration api_version=(2, 5, 0) to skip auto check_version requests on startup
[2025-09-04T01:09:01.843+0000] {consumer.py:118} WARNING - group_id is None: disabling auto-commit.
[2025-09-04T01:09:01.843+0000] {subscription_state.py:171} INFO - Updating subscribed topics to: ('streaming_data',)
[2025-09-04T01:09:01.843+0000] {subscription_state.py:257} INFO - Updated partition assignment: [('streaming_data', 0), ('streaming_data', 1), ('streaming_data', 2)]
[2025-09-04T01:09:01.844+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=kafka:29092 <connecting> [IPv4 ('172.16.240.5', 29092)]>: connecting to kafka:29092 [('172.16.240.5', 29092) IPv4]
[2025-09-04T01:09:01.844+0000] {conn.py:410} INFO - <BrokerConnection node_id=1 host=kafka:29092 <connecting> [IPv4 ('172.16.240.5', 29092)]>: Connection complete.
[2025-09-04T01:09:01.844+0000] {conn.py:919} INFO - <BrokerConnection node_id=bootstrap-0 host=kafka:29092 <connected> [IPv4 ('172.16.240.5', 29092)]>: Closing connection. 
[2025-09-04T01:09:01.958+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=kafka:29092 <connecting> [IPv4 ('172.16.240.5', 29092)]>: connecting to kafka:29092 [('172.16.240.5', 29092) IPv4]
[2025-09-04T01:09:01.958+0000] {conn.py:410} INFO - <BrokerConnection node_id=1 host=kafka:29092 <connecting> [IPv4 ('172.16.240.5', 29092)]>: Connection complete.
[2025-09-04T01:09:01.958+0000] {conn.py:919} INFO - <BrokerConnection node_id=bootstrap-0 host=kafka:29092 <connected> [IPv4 ('172.16.240.5', 29092)]>: Closing connection. 
[2025-09-04T01:09:01.961+0000] {logging_mixin.py:137} INFO - Produced post 1: sunt aut facere repellat provi...
[2025-09-04T01:09:01.964+0000] {logging_mixin.py:137} INFO - Consumed from partition 0, key 1: sunt aut facere repellat provi...
[2025-09-04T01:09:11.973+0000] {logging_mixin.py:137} INFO - Produced post 2: qui est esse...
[2025-09-04T01:09:11.978+0000] {logging_mixin.py:137} INFO - Consumed from partition 0, key 1: qui est esse...
[2025-09-04T01:09:21.984+0000] {logging_mixin.py:137} INFO - Produced post 3: ea molestias quasi exercitatio...
[2025-09-04T01:09:21.986+0000] {logging_mixin.py:137} INFO - Consumed from partition 0, key 1: ea molestias quasi exercitatio...
[2025-09-04T01:09:31.998+0000] {logging_mixin.py:137} INFO - Produced post 4: eum et est occaecati...
[2025-09-04T01:09:32.000+0000] {logging_mixin.py:137} INFO - Consumed from partition 0, key 1: eum et est occaecati...
[2025-09-04T01:09:42.006+0000] {logging_mixin.py:137} INFO - Produced post 5: nesciunt quas odio...
[2025-09-04T01:09:42.008+0000] {logging_mixin.py:137} INFO - Consumed from partition 0, key 1: nesciunt quas odio...
[2025-09-04T01:09:52.018+0000] {logging_mixin.py:137} INFO - Produced post 6: dolorem eum magni eos aperiam ...
[2025-09-04T01:09:52.023+0000] {logging_mixin.py:137} INFO - Consumed from partition 0, key 1: dolorem eum magni eos aperiam ...
[2025-09-04T01:10:02.031+0000] {logging_mixin.py:137} INFO - Produced post 7: magnam facilis autem...
[2025-09-04T01:10:02.033+0000] {logging_mixin.py:137} INFO - Consumed from partition 0, key 1: magnam facilis autem...
[2025-09-04T01:10:12.042+0000] {logging_mixin.py:137} INFO - Produced post 8: dolorem dolore est ipsam...
[2025-09-04T01:10:12.044+0000] {logging_mixin.py:137} INFO - Consumed from partition 0, key 1: dolorem dolore est ipsam...
[2025-09-04T01:10:22.055+0000] {logging_mixin.py:137} INFO - Produced post 9: nesciunt iure omnis dolorem te...
[2025-09-04T01:10:22.057+0000] {logging_mixin.py:137} INFO - Consumed from partition 0, key 1: nesciunt iure omnis dolorem te...
[2025-09-04T01:10:32.067+0000] {logging_mixin.py:137} INFO - Produced post 10: optio molestias id quia eum...
[2025-09-04T01:10:32.071+0000] {logging_mixin.py:137} INFO - Consumed from partition 0, key 1: optio molestias id quia eum...
[2025-09-04T01:10:42.076+0000] {logging_mixin.py:137} INFO - Produced post 11: et ea vero quia laudantium aut...
[2025-09-04T01:10:42.080+0000] {logging_mixin.py:137} INFO - Consumed from partition 2, key 2: et ea vero quia laudantium aut...
[2025-09-04T01:10:52.086+0000] {logging_mixin.py:137} INFO - Produced post 12: in quibusdam tempore odit est ...
[2025-09-04T01:10:52.088+0000] {logging_mixin.py:137} INFO - Consumed from partition 2, key 2: in quibusdam tempore odit est ...
[2025-09-04T01:11:02.096+0000] {logging_mixin.py:137} INFO - Produced post 13: dolorum ut in voluptas molliti...
[2025-09-04T01:11:02.098+0000] {logging_mixin.py:137} INFO - Consumed from partition 2, key 2: dolorum ut in voluptas molliti...
[2025-09-04T01:11:12.108+0000] {logging_mixin.py:137} INFO - Produced post 14: voluptatem eligendi optio...
[2025-09-04T01:11:12.110+0000] {logging_mixin.py:137} INFO - Consumed from partition 2, key 2: voluptatem eligendi optio...
[2025-09-04T01:11:22.120+0000] {logging_mixin.py:137} INFO - Produced post 15: eveniet quod temporibus...
[2025-09-04T01:11:22.122+0000] {logging_mixin.py:137} INFO - Consumed from partition 2, key 2: eveniet quod temporibus...
[2025-09-04T01:11:32.133+0000] {logging_mixin.py:137} INFO - Produced post 16: sint suscipit perspiciatis vel...
[2025-09-04T01:11:32.135+0000] {logging_mixin.py:137} INFO - Consumed from partition 2, key 2: sint suscipit perspiciatis vel...
[2025-09-04T01:11:42.146+0000] {logging_mixin.py:137} INFO - Produced post 17: fugit voluptas sed molestias v...
[2025-09-04T01:11:42.150+0000] {logging_mixin.py:137} INFO - Consumed from partition 2, key 2: fugit voluptas sed molestias v...
[2025-09-04T01:11:45.846+0000] {conn.py:1096} ERROR - <BrokerConnection node_id=1 host=kafka:29092 <connected> [IPv4 ('172.16.240.5', 29092)]>: socket disconnected
[2025-09-04T01:11:45.846+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=kafka:29092 <connected> [IPv4 ('172.16.240.5', 29092)]>: Closing connection. KafkaConnectionError: Socket EVENT_READ without in-flight-requests
[2025-09-04T01:11:45.846+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=kafka:29092 <connected> [IPv4 ('172.16.240.5', 29092)]>: Closing connection. KafkaConnectionError: socket disconnected
[2025-09-04T01:11:45.847+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2025-09-04T01:11:45.847+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2025-09-04T01:11:45.847+0000] {future.py:79} ERROR - Fetch to node 1 failed: KafkaConnectionError: socket disconnected
[2025-09-04T01:11:45.848+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=kafka:29092 <connecting> [IPv4 ('172.16.240.5', 29092)]>: connecting to kafka:29092 [('172.16.240.5', 29092) IPv4]
[2025-09-04T01:11:45.848+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=kafka:29092 <connecting> [IPv4 ('172.16.240.5', 29092)]> returned error 111. Disconnecting.
[2025-09-04T01:11:45.848+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=kafka:29092 <connecting> [IPv4 ('172.16.240.5', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2025-09-04T01:11:45.849+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2025-09-04T01:11:45.898+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=kafka:29092 <connecting> [IPv4 ('172.16.240.5', 29092)]>: connecting to kafka:29092 [('172.16.240.5', 29092) IPv4]
[2025-09-04T01:11:45.898+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=kafka:29092 <connecting> [IPv4 ('172.16.240.5', 29092)]> returned error 111. Disconnecting.
[2025-09-04T01:11:45.898+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=kafka:29092 <connecting> [IPv4 ('172.16.240.5', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2025-09-04T01:11:45.899+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2025-09-04T01:11:45.950+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=kafka:29092 <connecting> [IPv4 ('172.16.240.5', 29092)]>: connecting to kafka:29092 [('172.16.240.5', 29092) IPv4]
[2025-09-04T01:11:45.950+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=kafka:29092 <connecting> [IPv4 ('172.16.240.5', 29092)]> returned error 111. Disconnecting.
[2025-09-04T01:11:45.950+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=kafka:29092 <connecting> [IPv4 ('172.16.240.5', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2025-09-04T01:11:45.950+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2025-09-04T01:11:46.050+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=kafka:29092 <connecting> [IPv4 ('172.16.240.5', 29092)]>: connecting to kafka:29092 [('172.16.240.5', 29092) IPv4]
[2025-09-04T01:11:46.050+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=kafka:29092 <connecting> [IPv4 ('172.16.240.5', 29092)]> returned error 111. Disconnecting.
[2025-09-04T01:11:46.050+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=kafka:29092 <connecting> [IPv4 ('172.16.240.5', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2025-09-04T01:11:46.050+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2025-09-04T01:11:46.202+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=kafka:29092 <connecting> [IPv4 ('172.16.240.5', 29092)]>: connecting to kafka:29092 [('172.16.240.5', 29092) IPv4]
[2025-09-04T01:11:46.202+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=kafka:29092 <connecting> [IPv4 ('172.16.240.5', 29092)]> returned error 111. Disconnecting.
[2025-09-04T01:11:46.203+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=kafka:29092 <connecting> [IPv4 ('172.16.240.5', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2025-09-04T01:11:46.203+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2025-09-04T01:11:46.352+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=kafka:29092 <connecting> [IPv4 ('172.16.240.5', 29092)]>: connecting to kafka:29092 [('172.16.240.5', 29092) IPv4]
[2025-09-04T01:11:46.353+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=kafka:29092 <connecting> [IPv4 ('172.16.240.5', 29092)]> returned error 111. Disconnecting.
[2025-09-04T01:11:46.353+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=kafka:29092 <connecting> [IPv4 ('172.16.240.5', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2025-09-04T01:11:46.353+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2025-09-04T01:11:46.555+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=kafka:29092 <connecting> [IPv4 ('172.16.240.5', 29092)]>: connecting to kafka:29092 [('172.16.240.5', 29092) IPv4]
[2025-09-04T01:11:46.555+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=kafka:29092 <connecting> [IPv4 ('172.16.240.5', 29092)]> returned error 111. Disconnecting.
[2025-09-04T01:11:46.555+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=kafka:29092 <connecting> [IPv4 ('172.16.240.5', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2025-09-04T01:11:46.555+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2025-09-04T01:11:46.906+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=kafka:29092 <connecting> [IPv4 ('172.16.240.5', 29092)]>: connecting to kafka:29092 [('172.16.240.5', 29092) IPv4]
[2025-09-04T01:11:46.907+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=kafka:29092 <connecting> [IPv4 ('172.16.240.5', 29092)]> returned error 111. Disconnecting.
[2025-09-04T01:11:46.907+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=kafka:29092 <connecting> [IPv4 ('172.16.240.5', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2025-09-04T01:11:46.907+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
